core/load.py

```diff
@@ -362,13 +362,16 @@ def wait_until_interrupt(virtual_machine_container_reset_factory_time_seconds, c
                 # start containers based on selected modules
                 start_containers(configuration)
             if not new_network_events_thread.is_alive():
-                return error(messages["interrupt_application"])
+                error(messages["interrupt_application"])
+                return True
```

Comment:
return in a while loop?

Reply:
Return error("") 
Is returning None

So we need to return True in failing cases

We can break the while loop and at the end return True
What say @dhirensr ?

Reply:
we can't use break there. sounds like  a use case for break?